.env.example

```diff
@@ -25,4 +25,5 @@ VITE_GOOGLE_CLIENT_ID=94663405566-1go7jlpd2ar9u9urbfirmtjv1bm0tcis.apps.googleus
 VITE_GOOGLE_AUTH_ENABLED="false"
 
 # The command displayed in the UI for starting a local Python runtime.
-VITE_RUNTIME_COMMAND="deno run --unstable-broadcast-channel --allow-all --env-file=./.env jsr:@runt/pyodide-runtime-agent@0.6.4"
+# For Groq integration, use local runtime with environment variables
+VITE_RUNTIME_COMMAND="cd ../runt && GROQ_API_KEY=your_key deno run --allow-all --env-file=.env packages/pyodide-runtime-agent/src/mod.ts"
```

Comment:
This can just be `deno run --allow-all --env-file=.env ../runt/packages/pyodide-runtime-agent/src/mod.ts`. You'll want to put your `GROQ_API_KEY` in your `.env` files.